test/e2e/validation_test.go

```diff
@@ -136,6 +137,36 @@ func (tc *testContext) getInstanceIP(instanceID string) (string, error) {
 	return "", errors.New("unable to find Windows Worker nodes")
 }
 
+// getNode returns a pointer to the node object associated with the external IP provided
+func (tc *testContext) getNode(externalIP string) (*corev1.Node, error) {
```

Comment:
I thought we had a better way of getting the node object that does not depend on the VM having an external IP. Please follow that.